docker/ldap_config.docker.py

```diff
@@ -13,3 +13,10 @@ def __getattr__(name):
     except:
       pass
   raise AttributeError
+
+def __dir__():
+  names = []
+  for config in _loaded_configurations:
+    for name in config.__dir__():
+      names.append(name)
```

Comment:
I think this can be simplified to:

```suggestion
    names.extend(config.__dir__())
```